src/types/Strings.d.ts

```diff
@@ -5,17 +5,17 @@
  */
 
 /**
- * 0x prefixed Hexadecimal String
+ * Hexadecimal String
  */
-export declare type HexString = `0x${string}`;
```

Comment:
This also doesn't work  the way I wanted; again, only works for string literals and that's not the purpose I wanted it for.